apps/adapt-essentials-asset-fields/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "adapt-essentials-asset-fields",
+  "private": true,
+  "version": "1.0.0",
+  "type": "module",
+  "scripts": {
+    "install-ci": "npm ci",
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "test": "echo 'No tests yet'",
+    "lint": "eslint src --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "format": "prettier src/**/*.{ts,tsx} --write",
+    "preview": "vite preview",
+    "deploy": "contentful-app-scripts upload --ci --bundle-dir ./build --organization-id ${DEFINITIONS_ORG_ID} --definition-id 3Jt0c7RHcmPsB3mJAQC4iP --token ${CONTENTFUL_CMA_TOKEN}",
```

Comment:
do you mind changing the definition-id here to 6RGLVmXYrGaIIRIda1HwAC 
This is the new app definition that will exist within the Contentful Apps organization.